plugin/types.go

```diff
@@ -195,6 +195,12 @@ type Readable interface {
 	Open(context.Context) (SizedReader, error)
 }
 
+// Writable is an entry that can have data appended to it.
```

Comment:
Hmm, are we sticking with append because that is the (currently) supported behavior?

Reply:
I need to update this help text.